app/models/security_activity.rb

```diff
@@ -1,52 +1,70 @@
 class SecurityActivity < ApplicationRecord
-  enum event_type: {
-    login: 0,
-    change_email: 1,
-    change_phone: 2,
-    change_password: 3, # pragma: allowlist secret
-  }
-
-  belongs_to :user
-
-  def self.login!(user, ip_address)
-    new(
-      event_type: :login,
-      user_id: user.id,
-      ip_address: ip_address,
-    ).save!
-  end
+  # TODO: decide which of these to show on the security page
+  EVENTS = [
+    # logging in
+    ACCOUNT_LOCKED = LogEntry.new(id: 5, name: :account_locked, require_user: true),
+    MANUAL_ACCOUNT_UNLOCK = LogEntry.new(id: 6, name: :manual_account_unlock, require_user: true),
 
-  def self.login_with!(user, oauth_application, ip_address)
-    new(
-      event_type: :login,
-      user_id: user.id,
-      oauth_application_id: oauth_application.id,
-      ip_address: ip_address,
-    ).save!
-  end
+    TWO_STEP_VERIFICATION_SUCCESS = LogEntry.new(id: 7, name: :two_step_verification_success, require_user: true),
+    TWO_STEP_VERIFICATION_FAILURE = LogEntry.new(id: 8, name: :two_step_verification_failure, require_user: true),
 
-  def self.change_email!(user, ip_address)
-    new(
-      event_type: :change_email,
-      user_id: user.id,
-      ip_address: ip_address,
-    ).save!
+    LOGIN_SUCCESS = LogEntry.new(id: 0, name: :login_success, require_user: true),
```

Comment:
I'm divided on whether to have a separate event for https://github.com/alphagov/govuk-account-manager-prototype/blob/msw/security-events/config/initializers/doorkeeper.rb#L420-L424, which has `require_application: true`, and do a migration to split the current login events into the two types - what do others think?